aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -73,10 +76,11 @@ module aptos_framework::block {
         vm: signer,
         epoch: u64,
         round: u64,
-        previous_block_votes: vector<bool>,
-        missed_votes: vector<u64>,
         proposer: address,
+        proposer_index_optional: vector<u64>,
         failed_proposer_indices: vector<u64>,
+        previous_block_votes: vector<bool>,
+        missed_vote_indices: vector<u64>,
```

Comment:
I think we can remove this now right? This was on added for tracking vote performances on chain

Reply:
well, not in this diff - I need to update the rewards formula first. :)

Reply:
Might as well in this PR but as a separate commit?

Reply:
sounds good